src/routes/api.php

```diff
@@ -51,7 +51,13 @@
             Route::apiResource('polls', 'API\PollController')->only(['update'])->shallow();
             Route::put('polls/{poll}/publish', 'API\PollController@publish')->name('polls.publish');
             Route::get('polls/{poll}/results', 'API\PollController@results')->name('polls.results');
-            Route::apiResource('polls.choices', 'API\ChoiceController')->only(['store'])->shallow();
+            Route::apiResource('polls.choices', 'API\ChoiceController')->only(['store', 'update'])->shallow();
+        });
+
+        Route::apiResource('polls.choices', 'API\ChoiceController')->only(['index'])->shallow();
+
+        Route::middleware(['poll.role:' . App\User\Role::PRESENTER() . '|' . App\User\Role::ATENDEE()])->group(function () {
```

Comment:
params should be separed by a `,` and not a `|` c.f previous comment

Reply:
same answer